src/proto.c

```diff
@@ -287,53 +287,6 @@ void proto_close(int prot_sock)
 	proto->close(prot_sock);
 }
 
-int proto_mknode(int proto_socket, const char *owner_uuid,
-		 const char *device_name, const char *device_id,
-		 char *uuid, char *token)
-{
-	json_object *device;
-	const char *device_as_string;
-	json_raw_t response;
-	int err, result;
-
-	memset(&response, 0, sizeof(response));
-	device = parser_device_json_create(owner_uuid, device_name, device_id);
```

Comment:
This function is not been used anymore. Maybe you should call it in another place or remove it.